py_gtfs_rt_ingestion/tests/test_gtfs_rt_converter.py

```diff
@@ -9,20 +10,45 @@
 TEST_FILE_DIR = os.path.join(os.path.dirname(__file__), "test_files")
 
 
-def test_bad_conversion() -> None:
+def test_bad_conversion_local() -> None:
     """
-    test that bad filenames are returned back as the same strings. (these
-    filepaths are going to be moved from incoming to error.)
+    test that a bad filename will result in an empty table and the filename will
+    be added to the error files list
     """
     # dummy config to avoid mypy errors
     converter = GtfsRtConverter(config_type=ConfigType.RT_ALERTS)
 
-    bad_return = converter.gz_to_pyarrow(filename="badfile")
-    assert bad_return is None
+    # convert should still return a single element list of prefix, table tuple
+    tables = converter.convert(["badfile"])
+    assert len(tables) == 1
+
+    # archive files should be empty, error files should have the bad file in it
+    assert len(converter.archive_files) == 0
+    assert converter.error_files == ["badfile"]
+
+    # get the table and ensure its empty
+    _, table = tables[0]
+    assert table.num_rows == 0
+
 
+def test_bad_conversion_s3() -> None:
+    """
+    test that a bad s3 filename will result in an empty table and the filename
+    will be added to the error files list
+    """
+    converter = GtfsRtConverter(config_type=ConfigType.RT_ALERTS)
     with patch("pyarrow.fs.S3FileSystem", return_value=fs.LocalFileSystem):
-        bad_return = converter.gz_to_pyarrow(filename="s3://badfile")
-    assert bad_return is None
+        tables = converter.convert(["s3://badfile"])
+
+    assert len(tables) == 1
+
+    # archive files should be empty, error files should have the bad file in it
+    assert len(converter.archive_files) == 0
+    assert converter.error_files == ["s3://badfile"]
+
+    # get the table and ensure its empty
+    _, table = tables[0]
+    assert table.num_rows == 0
```

Comment:
split this tests into two tests. one for local one for s3.

now check that error files and archive files lists are correct and that convert produces empty tables with no rows.